benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1824,12 +1826,12 @@ void GraphicsBenchmarkApp::RecordCommandBufferFullscreenQuad(PerFrame& frame, si
             float3 colorValues = pFullscreenQuadsColor->GetValue();
             colorValues *= intensity;
             mQuadPushConstant.ColorValue = colorValues;
-            frame.cmd->PushGraphicsConstants(mQuadsPipelineInterfaces[0], GetPushConstCount(mQuadPushConstant.ColorValue), &mQuadPushConstant.ColorValue, offsetof(QuadPushConstant, ColorValue) / sizeof(uint32_t));
+            frame.cmd->PushGraphicsConstants(mQuadsPipelineInterfaces[1], GetPushConstCount(mQuadPushConstant.ColorValue), &mQuadPushConstant.ColorValue, offsetof(QuadPushConstant, ColorValue) / sizeof(uint32_t));
```

Comment:
nice catch, maybe it would be clearer to use FullscreenQuadsType::FULLSCREEN_QUADS_TYPE_SOLID_COLOR here instead of 1 or maybe pFullscreenQuadsType->GetValue()? (same as the other places you've fixed)